package.json

```diff
@@ -18,13 +18,14 @@
         "lint": "eslint --fix './src/**/*.ts'",
         "format": "prettier --write 'src/**/*.ts'",
         "build": "npm run clean && npm run lint && npm run format && tsc",
-        "docker": "docker-compose up --force-recreate vault &",
+        "docker": "docker-compose up --force-recreate -d",
+        "postdocker": "node dist/tests/integration/bootstrap.js",
         "docker:kill": "docker-compose kill",
         "pretest": "npm run build && npm run docker",
         "posttest": "npm run docker:kill",
-        "test": "wait-on --timeout 20000 http://localhost:8201 && npm run lint && npm run test:unit && npm run test:integration",
+        "test": "wait-on --timeout 20000 http://localhost:8200 && npm run lint && npm run test:unit && npm run test:integration",
```

Comment:
8201 was the port of the custom python server the old image provided to retrieve the client token. Since that's gone, I changed to the vault port for the readiness check.